test/helpers/cilium.go

```diff
@@ -420,6 +421,8 @@ func (s *SSHMeta) PolicyGetRevision() (int, error) {
 // PolicyImportAndWait imports a new policy into Cilium and waits until the policy
 // revision number increments.
 func (s *SSHMeta) PolicyImportAndWait(path string, timeout time.Duration) (int, error) {
+	ginkgo.By(fmt.Sprintf("Setting up policy: %s", path))
```

Comment:
I need to say that I don't  like to have By on helpers. It's not a proper sub-test into the test, and it's not into the test itself. 

My idea, if you are checking the `By`, you will expect to have it in the test, in the helpers is a bit weird for me

Happy to keep in there, but a second opinion will be great :-)

Reply:
I haven't really thought of `By()` as being a sub-test, but rather just marking important steps through the test. When debugging a policy failure in non-verbose mode, having a line on the commandline that states `STEP: Setting up policy: <path>` is useful because then I can immediately go from the test failure output to the relevant policy and try to figure out which part of the policy failed. I don't need to scan through the test, figure out what it's doing, figure out where the policy path is, then dig into it.

On the flip side, in general I think it's nice to have the `By()` expressions directly in the test files rather than the helpers, because (at least in my editor) the strings are highlighted in a bright colour and are easy to pick out, which breaks the tests down into readable sections. This approach doesn't do this for important steps like policy modification or restarting Cilium.

IMO the best middle ground for `By()` statements in helpers is to use them sparingly there, and prefer them to exist in the tests. These particular cases I like, so I plan to leave them in place unless someone feels strongly that they should not live there.

Reply:
Make sense to me :-) Thanks!